### PR TITLE
[1.13] A sweep through deprecations

### PR DESCRIPTION
**Pull request by williewillus**
Low risk cleanup commit, just sweeping through and removing everything that's clearly marked "remove 1.13" or "only here for binary compat"

### CODE REVIEW

**Comment by Shadows-of-Fire** [comment]
The old forge ItemTesr stuff should be deleted if it still exists, as it was replaced by teisr and was deprecated since 1.9

**Comment by tterrag1098** [comment]
I'm pretty sure I skipped those patches

File: src/main/java/net/minecraftforge/fml/common/gameevent/PlayerEvent.java [hunk 1]
```diff
@@ -34,34 +34,6 @@ private PlayerEvent(EntityPlayer player)
         this.player = player;
     }
 
-    public static class ItemPickupEvent extends PlayerEvent {
```
Comment (williewillus) [inline src/main/java/net/minecraftforge/fml/common/gameevent/PlayerEvent.java]: note: this FML item pickup event isn't marked deprecated, but it's been duplicated by the forge item pickup event for a while. Does anyone still need something from this FML event that the Forge one doesn't let you do?

Comment (bs2609) [reply src/main/java/net/minecraftforge/fml/common/gameevent/PlayerEvent.java]: The comments in #3419 suggest that the two events aren't exactly equivalent.

(e.g. https://github.com/MinecraftForge/MinecraftForge/issues/3419#issuecomment-307020636, https://github.com/MinecraftForge/MinecraftForge/issues/3419#issuecomment-324213982)

Comment (tterrag1098) [reply src/main/java/net/minecraftforge/fml/common/gameevent/PlayerEvent.java]: Should probably just leave it for a later commit to clean up both events.

**Comment by williewillus** [comment]
put the old itempickupevent back, should be ready

**Pull request merged**